content/en/docs/collector/_index.md

```diff
@@ -40,4 +39,16 @@ can take care of additional handling like retries, batching, encryption or even
 
 It is also easier to [setup a collector](./getting-started) than you might think: the default OTLP exporters in each language assume a local collector endpoint, so you'd start up a collector and you'd just start getting telemetry.
 
+## Status and releases
+
+The **collector** status is: [Stable][].
```

Comment:
The collector has not yet reached stable status, as not all of its core components are stable.

Reply:
Thanks. What would be the correct status?

Reply:
Maybe the current status might be considered as [**mixed**](https://opentelemetry.io/docs/reference/specification/document-status/#mixed)?

Reply:
> The collector has not yet reached stable status, as not all of its core components are stable.

I've updated the status to "mixed", which is an official spec term :). PTAL.